.github/workflows/Documentation.yml

```diff
@@ -41,5 +41,5 @@ jobs:
       - name: BuildAndDeploy
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }}
+          # DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }}
```

Comment:
Looks like Documenter would check `DOCUMENTER_KEY` first, but since you didn't provide the ssh key for it, it's fine to just use GITHUB_TOKEN, which is generated by Github Action automatically.